src/main/java/core/service/CreateReportToCsv.java

```diff
@@ -0,0 +1,20 @@
+package core.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CreateReportToCsv implements CreateReportToFile {
```

Comment:
```suggestion
public class CsvFileWriterService implements FileWriterService {
```
what about this comment?